modules/android/pages/p2psync-multipeer.adoc

```diff
@@ -4,347 +4,656 @@ ifdef::show_edition[:page-edition: Enterprise Edition]
 
+For small mesh topologies, Multipeer P2P Replicator offers `autodiscovery` for wifi-based networks and secure communication via TLS and certificate-based authentication.
+The dynamic mesh topology gives optimal peer connectivity and the lightweight and low-maintenance configuration requires less management and less code than using Active-Passive peer-to-peer sync.
 
-// Bluetooth - not yet
 
 [TIP]
 .Bluetooth networking
 ====
-Multipeer Replication is currently available for use over IP networks.
+Multipeer Replication is available for use over IP networks.
 If you are working over a bluetooth network, then take a look at xref:p2psync-custom.adoc[integrating a Custom Listener]
 with Couchbase Lite's Active-Passive Peer-to-Peer synchronization.
 ====
 
 
+== Overview
+
+
+The Multipeer P2P Replicator automatically advertises its presence, discovers other peers,
+and establishes connections for data replication with peers that share the same group identifier defined by the application.
+
+
+To maintain optimal connectivity, efficient data transport, and balanced workloads, the Multipeer P2P Replicator forms a dynamic mesh network among peers in the same group.
+
+
+It avoids redundant direct connections, evenly distributes connections across peers, and optimizes communication paths.
+The mesh network continuously adapts as peers join or leave.
 
-// end::introduction[]
-// end::introduction-full[]
 
+// Mesh Diagram
+[mermaid]
+....
+flowchart TB;
+   id1[Android Device #1]
+   id2[Android Device #2]
+   id3[Android Device #3]
+   id4[Android Device #4]
+   id5[Android Device #5]
+   id6[Android Device #6]
+   id1<-->id2
+   id1<-->id3
+   id1<-->id4
+   id1<-->id5
+   id1<-->id6
+   id2<-->id3
+   id2<-->id4
+   id2<-->id5
+   id2<-->id6
+   id3<-->id4
+   id3<-->id5
+   id3<-->id6
+   id4<-->id5
+   id4<-->id6
+   id5<-->id6
+....
 
 
 == Prerequisites
 
+
 === Transport and Peer Discovery Protocol
 
+
 Multipeer P2P Replicator supports Wi-Fi transport, using `DNS-SD` (also known as Bonjour) for peer discovery.
-Peers must be connected to the same Wi-Fi network to discover and establish connections with one another.
+Peers connect to the same Wi-Fi network to discover and establish connections with one another.
+
 
 === Supported Platforms
 
-For Android, we recommend using a recent release -- preferably supporting API 34, or more recent -- but older versions should work with the multipeer sync feature.
 
-// Below more applicable with Bluetooth - review and uncomment if/when needed
-// For Android, the minimum API level is API 35, based on the availability of required DNS-SD features for peer discovery and mesh network construction—such as support for subtypes, and the ability to publish and subscribe to service information.
+For Android, we recommend using a recent release, preferably supporting minimum API level 24, or more recent but earlier versions should work with the multipeer sync feature.
```

Comment:
Are we supporting earlier versions than 24?
Confirm with PM if we're saying  they should work but are or are not supported?